pkg/storage/article_storage.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"news-aggregator/pkg/model"
+	"time"
+)
+
+//go:generate mockgen -destination=mocks/mock_article.go -package=mocks news-aggregator/pkg/storage ArticleStorage
+
+// ArticleStorage is an interface that defines the methods for interacting with the article storage.
+type ArticleStorage interface {
+	GetAll() ([]model.Article, error)
```

Comment:
instead of this `model.Article` struct, try to have more abstract type to be able to store any data.

Reply:
As I mentioned before, this type of storage used for specified type of model, as Article